app/Http/Kernel.php

```diff
@@ -60,6 +60,8 @@ class Kernel extends HttpKernel
         'signed' => \Illuminate\Routing\Middleware\ValidateSignature::class,
         'throttle' => \Illuminate\Routing\Middleware\ThrottleRequests::class,
         'verified' => \Illuminate\Auth\Middleware\EnsureEmailIsVerified::class,
+
+        'reminder.is.valid' => \App\Http\Middleware\ReminderIsValid::class,
```

Comment:
This is pretty much unnecessary by the same reasons as above: Laravel handles this by itself.
Plus, concurrency won't be an issue for the same user scope (this isn't a multi tenant app) - as such, Reminders won't be accessed when non-existent.